test/support/_shutdown_ctx_mgr.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
```

Comment:
is this file used?

Reply:
(referring to `_shutdown_ctx_mgr.py` not `asyncio`)

Reply:
Yup, in the new tests `test_shutdown_during_ctx_mgr_setup` and `test_shutdown_during_ctx_mgr_yield` in shutdown_tests.py